src/sheet_metal_conus.py

```diff
@@ -3,188 +3,338 @@
         self.OptionParser.add_option('-b', '--diaBase', action = 'store',
-        type = 'float', dest = 'diaBase', default = 300.0000,
-        help = 'The diameter of the cones base.')
+            type = 'float', dest = 'diaBase', default = 300.0,
+            help = 'The diameter of the cones base.')
 
         self.OptionParser.add_option('-c', '--diaCut', action = 'store',
-        type = 'float', dest = 'diaCut', default = 100.0000,
-        help = 'The diameter of cones cut (0.0 if cone is not cut.')
+            type = 'float', dest = 'diaCut', default = 100.0,
+            help = 'The diameter of cones cut (0.0 if cone is not cut.')
 
         self.OptionParser.add_option('-l', '--heightCone', action = 'store',
-        type = 'float', dest = 'heightCone', default = 200.0000,
-        help = 'The height of the (cut) cone.')
+            type = 'float', dest = 'heightCone', default = 200.0,
+            help = 'The height of the (cut) cone.')
         
         self.OptionParser.add_option('-u', '--units', action = 'store',
-        type = 'string', dest = 'units', default = 'millimetre',
-        help = 'The units in which the cone values are given. Can be millimetre or inch.')
+            type = 'string', dest = 'units', default = 'mm',
+            help = 'The units in which the cone values are given. mm or in for real objects')
         
         self.OptionParser.add_option('-w', '--strokeWidth', action = 'store',
-        type = 'float', dest = 'strokeWidth', default = 0.4,
-        help = 'The line thickness in given unit. For laser cutting it should be rather small.')
+            type = 'float', dest = 'strokeWidth', default = 0.3,
+            help = 'The line thickness in given unit. For laser cutting it should be rather small.')
 
         self.OptionParser.add_option('-f', '--strokeColour', action = 'store',
-        type = 'string', dest = 'strokeColour', default = 'blue',
-        help = 'The line colour. Can be given in numbers (e.g. \'#000000\') or by name (e.g. \'blue\').')
+            type = 'string', dest = 'strokeColour', default = 896839168, # Blue
+            help = 'The line colour.')
 
         self.OptionParser.add_option('-d', '--verbose', action = 'store',
-        type = 'inkbool', dest = 'verbose', default = False,
-        help = 'Enable verbose output of calculated parameters. Used for debugging or is someone needs the calculated values.')
+            type = 'inkbool', dest = 'verbose', default = False,
+            help = 'Enable verbose output of calculated parameters. Used for debugging or is someone needs the calculated values.')
 
-    def effect(self):
-        """
-        Effect behaviour.
-        Overrides base class' method and draws rolled out sheet metal cone into SVG document.
-        """
-        if self.options.units == 'inch':
-                SheetMetalConus.convFactor=SheetMetalConus.inch2pixel
 
-        # Store all the relevants values in a dictionary for easy access
```

Comment:
still using same approach here, but moved lower down and some unused vars removed